rbac/management/role/relation_api_dual_write_handler.py

```diff
@@ -40,6 +41,151 @@
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+class SeedingRelationApiDualWriteHandler:
+    """Class to handle Dual Write API related operations specific to the seeding process."""
+
+    _replicator: RelationReplicator
+    _current_role_relations: list[common_pb2.Relationship]
+
+    _public_tenant: Optional[Tenant] = None
+    _platform_default_policy_uuid: Optional[str] = None
+    _admin_default_policy_uuid: Optional[str] = None
+
+    def __init__(self, replicator: Optional[RelationReplicator] = None):
+        """Initialize SeedingRelationApiDualWriteHandler."""
+        self._replicator = replicator if replicator else OutboxReplicator()
+
+    def replication_enabled(self):
```

Comment:
I wonder if we can make a DualWriteHandler class for inheritance to reuse some of the code. I saw these in several places